app/database/operator/server_data_operator/handlers/post.test.ts

```diff
@@ -169,13 +169,15 @@ describe('*** Operator: Post Handlers tests ***', () => {
                 metadata: {},
             },
         ];
-    });
 
-    beforeAll(async () => {
```

Comment:
This change is to make sure tests have independent databases, and the result of one test don't pollute the data of others.